banks-interface/src/lib.rs

```diff
@@ -2,20 +2,22 @@
 
 use {
     serde_derive::{Deserialize, Serialize},
-    solana_sdk::{
-        account::Account,
-        clock::Slot,
-        commitment_config::CommitmentLevel,
-        hash::Hash,
-        inner_instruction::InnerInstructions,
-        message::Message,
-        pubkey::Pubkey,
-        signature::Signature,
-        transaction::{self, TransactionError, VersionedTransaction},
-    },
+    solana_account::Account,
+    solana_clock::Slot,
+    solana_commitment_config::CommitmentLevel,
+    solana_hash::Hash,
+    solana_message::{inner_instruction::InnerInstructions, Message},
+    solana_pubkey::Pubkey,
+    solana_signature::Signature,
+    solana_transaction::versioned::VersionedTransaction,
     solana_transaction_context::TransactionReturnData,
+    solana_transaction_error::TransactionError,
 };
 
+pub mod transaction {
+    pub use solana_transaction_error::TransactionResult as Result;
+}
+
```

Comment:
i did this in several places to keep to minimize touch points outside use statements

Reply:
Does this need to be `pub` ?

Reply:
derp.. not the mod :facepalm: